onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -183,10 +183,12 @@ bool VariadicNodeGroupSelector::Check(const GraphViewer& graph_viewer,
   return dt_input == dt_output;
```

Comment:
It is assumed only input can be variadic. Please also check output in same way as input, i.e., using a for loop